meritoo/common-bundle/0.1/config/packages/meritoo_common.yaml

```diff
@@ -0,0 +1,7 @@
+#
+# All configuration options are described here:
+# https://github.com/meritoo/common-bundle/blob/master/docs/Configuration.md
```

Comment:
the could be a one liner

Reply:
In this case line will be longer than 120, so it's better to use 2 lines